spec/lib/administrate/order_spec.rb

```diff
@@ -36,7 +36,7 @@
 
         ordered = order.apply(relation)
 
-        expect(relation).to have_received(:order).with(name: :asc)
+        expect(relation).to have_received(:order).with("name asc")
```

Comment:
Yeah, I think I'm with you on that. I was curious to see if we were testing the old option as well as the new, but I think we're good.